homeassistant/components/fritz/binary_sensor.py

```diff
@@ -17,79 +22,89 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+def _retrieve_connection_state(status: FritzStatus) -> bool:
+    """Return download line attenuation."""
+    return bool(status.is_connected)
+
+
+def _retrieve_link_state(status: FritzStatus) -> bool:
+    """Return download line attenuation."""
+    return bool(status.is_linked)
+
+
+class SensorData(TypedDict):
```

Comment:
Please use `SensorEntityDescription` instead of this.

Reply:
Done